packages/neo-one-client-common/src/crypto.ts

```diff
@@ -22,11 +22,10 @@ import { p256 } from './precomputed';
 import { ScriptBuilder } from './ScriptBuilder';
 
 // tslint:disable-next-line no-let
-let ecCache: any;
+let ecCache: EC | undefined;
 const ec = () => {
   if (ecCache === undefined) {
-    // tslint:disable-next-line no-any
-    ecCache = new EC(p256) as any;
+    ecCache = new EC(p256);
```

Comment:
Could this be simplified by assigning ` new EC(p256) ` as the default ` ecCache `:

```typescript
let ecCache: EC = new EC(p256);
const ec = () => ecCache;
```

Reply:
no, we specifically do it this way so that `ec` is a singleton not created until something needs it. The changes I made here were only because EC is typed now and we don't need the `any`.

Reply:
LGTM